.github/comment.yml

```diff
@@ -74,8 +74,10 @@
 - rule:
     type: label
     label: ARMChangesRequested
+    onLabeledComments: "Hi, @${PRAuthor} Your PR has been labeled with `ARMChangesRequested`, please update PR accroding to comments. When you resolve all issues, please remove `ARMChangesRequested` label, bot will add "WaitForARMFeedback"  and back to ARM review process."
```

Comment:
... please ensure to respond feedbacks from the ARM API reviewer.  When you are ready to continue the ARM API review, please remove 'ARMChangesRequested' ...